doc/release-notes/5.3-release-notes.md

```diff
@@ -132,27 +132,37 @@ Please configure your connection details, replacing all the `${DB_...}`.
 - `<payara install path>/bin/asadmin create-password-alias --passwordfile /tmp/password.txt dataverse.db.password`
 - `rm /tmp/password.txt`
 
-4\. Reset the EJB timer database back to default:
+4\. Replace relevant portions of domain.xml
```

Comment:
this step should be optional for those who are coming from what was originally a glassfish4 domain.xml. (and if we also do add this to the 5.0 release notes, per the multiple server issue from the mailing list, we may want text like (if you haven't already...). we can discuss this in person, if you'd like to understand this better - the whys and whens.

Reply:
@scolapasta sure, I'll take you up on that. It'll be good to talk about this.